main.cpp

```diff
@@ -8,10 +8,10 @@
 #include "GPIO.h"
 #include "Test.h"
 #include "GpioTesting.h"
-
+#include "LogicLayer.h"
+#include "HAL.h"
```

Comment:
wenn du HAL noch in HardwareLayer umbenennen könntest wäre das ziemlich cool.
Wenn der Code einmal kompiliert kann man das ja mit dem Refactoring Tool machen. Allerdings musst du danach auch noch die Dateinamen ändern und dann auch die #includes an alles stellen. Am besten danach noch einmal kompilieren lassen.

Reply:
done